lib/bolt/config.rb

```diff
@@ -334,7 +334,7 @@ def validate
         raise Bolt::ValidationError, "Unsupported format: '#{format}'"
       end
 
-      Bolt::Util.validate_file('hiera-config', @data['hiera_config']) if @data['hiera_config']
+      Bolt::Util.validate_file('hiera-config', @data['hiera-config']) if @data['hiera-config']
```

Comment:
Does this imply we were previously never validating hiera-config?

Reply:
I believe so! Since this PR will be blocked for a while, it might be worth getting this in as a bug fix separately.